### ISSUE TITLE
Topic Outline & contributors 

### ISSUE THREAD

**Issue by MWins** [taniarascia/startmakingwebsites#2]
Might help if there was an outline of topics. As is, not sure exactly what topics are considered in scope for the guide. Is HTTP going to be covered? Will XML/JSON ?

Need some system set up for contributors to submit sections without editing index.html. Maybe a src/ directory with fragments and those get concatenated into index.html ?

**Comment by taniarascia** [comment]
You're absolutely right, I'll need to work on that. I definitely don't want the index to be enormous, so I'll start breaking it down into sections. I'll work on a README covering the topics, and create a better way to use Markdown and contribute. Thanks for your contributions!

**Comment by MWins** [comment]
I'm not sold on Markdown as best for your project. It does a great job of handling text content which has simple needs. Not sure it's going to be ideal. I'm looking at other options. 

Also,  consider adding a link to the website on the front of the github repo.

**Comment by taniarascia** [comment]
Yep, I'm definitely going to make a Readme. I was looking into using Jekyll for static website hosting. I haven't used Jekyll yet, but it looks like you can easily use it for static sites, not just blogs. In that case, I would assume each source file can be compiled into a layout with the header, nav and footer, and then each section would have its own page.

**Comment by taniarascia** [comment]
I made [this page](http://taniarascia.github.io/markdown/) as an example of how the site would work with Markdown. [Source](https://github.com/taniarascia/markdown). I think it works quite well!

**Comment by MWins** [comment]
It looks good. I'll probably write up XML/JSON short pieces for when you get to AJAX.
